js/reference.js

```diff
@@ -5401,6 +5401,36 @@
             return this._isEntityMode;
         },
 
+        /**
+         * Returns true if the plotly histogram graph should be shown in the UI
+         * If _facetObject.barPlot is not defined, the value is true. By default
+         * the histogram should be shown unless specified otherwise
+         *
+         * @type {Boolean}
+         */
+        get showHistogram() {
```

Comment:
I think it's better if we change the name of the api to be `barPlot` (same as the attribute name). Also the name of attributes in the annotation are usually snake case and not camel case. I think we should change barPlot to bar_plot and nBins to n_bins to be consistent.